### PR TITLE
SE-34&35: User Profile + Video History pages

### PR DESCRIPTION
**Pull request by uppala-supriya**
Updated code for user profile and video history .

### CODE REVIEW

**Comment by deepalii17** [comment]
Please update the PR title to be of the format mentioned [here](https://docs.google.com/document/d/1eC5MVZNIDHit2Us3kN203WJLTzY9hh4Q1nLN55sbkoc/edit?usp=sharing)

File: code/frontend/src/App.js [hunk 1]
```diff
@@ -28,6 +30,8 @@ function App() {
         <Route path='/' element={screenToShow()} />
         <Route path='/search' element={<Search />} />
         <Route path='/browse' element={<EducationalVideoFeed />} />
+        <Route path='/userProfile' element={<UserProfile />} />
+        <Route path='/videoHistory' element={<VideoHistory />} />
```
Comment (deepalii17) [inline code/frontend/src/App.js:34]: NitPick: Please follow the naming convention for rest uris: https://restfulapi.net/resource-naming/

**Review by deepalii17 — CHANGES_REQUESTED** [review]

File: code/frontend/src/App.js (hunk 1 above)
Comment (uppala-supriya) [reply code/frontend/src/App.js]: > NitPick: Please follow the naming convention for rest uris: https://restfulapi.net/resource-naming/

changed and updated the code.

**Review by uppala-supriya — COMMENTED** [review]
followed RESTful naming conventions for your route paths and updaed

**Review by deepalii17 — APPROVED** [review]

**Pull request merged**